### DISCUSSION TITLE
How to change color of SSH terminal ?

### DISCUSSION THREAD

**Discussion by user858753257** [Eugeny/tabby#7201] [General]
I want to change color of a SSH shell. 

my own color scheme  only works on wsl etc...

![image](https://user-images.githubusercontent.com/8161283/193354269-8b8d2b7f-4864-4015-939d-b75c5bc64cd9.png)

**Comment by Eugeny** [comment]
https://github.com/Eugeny/tabby/discussions/5258

**Reply by user858753257** [reply]
thanks

**Reply by user858753257** [reply]
How can i do that onw win10 ?

**Comment by user858753257** [comment]
@Eugeny 

have anothe color question:
![image](https://user-images.githubusercontent.com/8161283/193356859-f3567153-0717-491d-818e-932e39652560.png)

How can i change the "Folder" color ? Which of the ANSI values i have to edit?
